core/snippets/snippets/importFrom.snippet

```diff
@@ -0,0 +1,14 @@
+name: importFromStatement
+phrase: import from
+
+---
+
+language: javascript | typescript | javascriptreact | typescriptreact
+-
+import { $0 } from "$1";
```

Comment:
From the community backlog session we recommended that you remove the braces around `$0`, and swap $0 and $1 (i.e., the token order you have in python is correct)

Reply:
Done